doc/external-libs/cuda.html

```diff
@@ -45,10 +47,12 @@ <h1>Installing deal.II with CUDA</h1>
       </pre>
       <code>-DDEAL_II_CUDA_FLAGS_DEBUG</code> and 
       <code>-DDEAL_II_CUDA_FLAGS_RELEASE</code> are also available if you want
-      a finer control on the CUDA flags. Finally, the CUDA compiler and the
+      a finer control on the CUDA flags. The CUDA compiler and the
       CUDA toolkit root directory can be set using 
       <code>-DDEAL_II_CUDA_COMPILER</code> and 
       <code>-DDEAL_II_CUDA_TOOLKIT_ROOT_DIR</code>.
+      Finally, the CUDA host compiler is the same as the C++ compiler
+      by default, but can be changed using CUDA flags as well.
```

Comment:
Have you tried that? You can compile deal with gcc 6 and have the CUDA host compiler to be gcc 5?

Reply:
Yes, I tested that I can set the `CUDA` host compiler to `gcc-8` while the C++ compiler is `gcc-5` using `-ccbin` even with this PR. Of course, this is not really useful, but if people want to use a different `CUDA` host compiler that is compatible with the flags set due to the C++ compiler, they can.